README.md

```diff
@@ -1,2 +1,11 @@
 # Hello-Microverse
 This repo is created for Day 2 activity of Microverse Full-Time web developer course.
+
+The Milestones of this project:
+#1 Creating new feature branch to work on it
+#2 Adding ".gitignore" file with arguments
+#3 Setting up a linter for HTML&CSS enviroment
+#4 Adding descriptive "README" file
+#5 Adding a HTML and a Css file
+#6 Adding .md file for testing to ignore it
+#7 Open Pull request
```

Comment:
- [ ] You did a good job implementing the README file :+1: but as a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should customize it with this [template](https://github.com/microverseinc/readme-template). It would make your project looks more professional and this is one of the first things that hiring managers or your potential colleagues will check when considering you for a position on their team. To learn more you could read [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md)